src/assets/scripts/client/scope/ScopeModel.js

```diff
@@ -301,6 +301,10 @@ export default class ScopeModel {
         if (scratchPadText.length > 3) {
             return [false, 'ERR: SCRATCHPAD MAX 3 CHAR'];
         }
+        
+        if (scratchPadText === '---') {
+            return radarTargetModel.setScratchpad('');
+        }
```

Comment:
Let's change this to use a single period mark to clear the scratchpad. This is how it is done in vSTARS, and thus will be easier for those users to adjust to, while being no more complicated than the `---` (but with the added benefit of being fewer keystrokes).

Reply:
```suggestion
        if (scratchPadText === '.') {
            return radarTargetModel.setScratchpad('');
        }
```

Would this be the desired implementation then?

I'll check on the docs and update them as necessary. Also going to work on moving this to an openscope branch as I don't think it was.

Reply:
yep, exactly.